integrated_channels/canvas/models.py

```diff
@@ -123,8 +123,8 @@ class CanvasLearnerDataTransmissionAudit(models.Model):
     )
 
     enterprise_course_enrollment_id = models.PositiveIntegerField(
-        blank=False,
-        null=False,
+        blank=True,
```

Comment:
is this needed because we won't always have the enrollment id when we create this record?if so wil something later update this id field here?

Reply:
You know what, when I made this PR I was looking at the other channels models and saw that some of them set this value to not required. At the time I was assuming that the learner data transmission audit would be intantiated earlier in the process. As of right now, I don't believe this model is generated until a learner has enrolled in the course. If the user is enrolled then there should be an enterprise course enrollment id.. right? Maybe this PR isn't actually needed? Unless you can think why a channel like cornerstone has these values set to nullable?

Reply:
hmm, not clear how we use this field, is it required to send grade passback? if yes then we may want to at least go with null=False. The blank is just for frontend or forms so not too worried

yeah if this field must always be set (and not setting it should be an error), then null=False is what we want